scripts/import_templates_gen3.py

```diff
@@ -0,0 +1,131 @@
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Script for copying templates appropriate for these fields.
+
+The datasets can be from any source, including the generate_templates_gen3.sh
+script in this directory.
+
+Example:
+$ python import_templates_gen3.py -t "u/me/DM-123456-template"
+imports goodSeeing templates from u/me/DM-123456-template in /repo/main to
+templates/goodSeeing in this dataset's preloaded repo. See
+generate_templates_gen3.sh -h for more options.
+"""
+
+import argparse
+import logging
+import os
+import sys
+import tempfile
+
+import lsst.log
+import lsst.skymap
+from lsst.daf.butler import Butler, CollectionType
+
+
+logging.basicConfig(level=logging.INFO, stream=sys.stdout)
+lsst.log.configure_pylog_MDC("DEBUG", MDC_class=None)
+
+
+########################################
+# Command-line options
+
+def _make_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-b", dest="src_dir", default="/sdf/group/rubin/repo/main",
+                        help="Repo to import from, defaults to '/sdf/group/rubin/repo/main'.")
+    parser.add_argument("-t", dest="src_collection", required=True,
+                        help="Template collection to import from.")
+    return parser
+
+
+args = _make_parser().parse_args()
+
+
+########################################
+# Export/Import
+
+TEMPLATE_TYPE = "goodSeeing"
+TEMPLATE_NAME = TEMPLATE_TYPE + "Coadd"
+TEMPLATE_COLLECT = "templates/" + TEMPLATE_TYPE
+SCRIPT_DIR = os.path.abspath(os.path.dirname(__file__))
+DATASET_REPO = os.path.normpath(os.path.join(SCRIPT_DIR, "..", "preloaded"))
+
+def _export(butler, export_file):
+    """Export the files to be copied.
+
+    Parameters
+    ----------
+    butler : `lsst.daf.butler.Butler`
+        A Butler pointing to the repository and collection(s) to be
+        exported from.
+    export_file : `str`
+        A path pointing to a file to contain the export results.
+
+    Returns
+    -------
+    runs : iterable [`str`]
+        The names of the runs containing exported templates.
+    """
+    skymaps = butler.registry.queryDataIds("skymap", datasets=TEMPLATE_NAME, collections=butler.collections)
+    unique_skymaps = set([id for id in skymaps])
```

Comment:
There's no need to convert to a `list` first. I suggest either:
```suggestion
    skymaps = butler.registry.queryDataIds("skymap", datasets=TEMPLATE_NAME, collections=butler.collections)
    unique_skymaps = {id for id in skymaps}
```
or
```suggestion
    skymaps = set(butler.registry.queryDataIds("skymap", datasets=TEMPLATE_NAME, collections=butler.collections))
```